internal/store/clusters.go

```diff
@@ -62,10 +62,10 @@ func (s *Store) CreateCluster(cluster *atlas.Cluster) (*atlas.Cluster, error) {
 }
 
 // UpdateCluster encapsulate the logic to manage different cloud providers
-func (s *Store) UpdateCluster(cluster *atlas.Cluster) (*atlas.Cluster, error) {
+func (s *Store) UpdateCluster(projectID, name string, cluster *atlas.Cluster) (*atlas.Cluster, error) {
```

Comment:
when pausing/starting a cluster we can only submit, the `Paused` property or they API will error with a "can't update and pause at the same time" even if the values are not changing so we needed to reconfigure this